src/Scenario/Processor.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Scenario;
+
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\Package\Manager;
+use Magento\MagentoCloud\Process\ProcessInterface;
+use Magento\MagentoCloud\Scenario\Exception\ProcessorException;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Process given scenarios.
+ */
+class Processor
+{
+    /**
+     * @var Merger
+     */
+    private $merger;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var Manager
```

Comment:
Manager of what?
Add more accurate description.
The name of the property is also better to rename, e.g. $packageManager

Reply:
Renamed property

Reply:
Renamed property